code/game/objects/items/storage/fancy.dm

```diff
@@ -155,6 +155,16 @@
 	. = ..()
 	atom_storage.set_holdable(list(/obj/item/food/egg))
 
+/*
+ * Fertile Egg Box
+ */
+
+/obj/item/storage/fancy/egg_box/fertile
+	name = "fertile egg box"
+	desc = "Only one thing here is fertile, and it's not the eggs."
```

Comment:
🤨

Reply:
new pr from the person behind the people milker

Reply:
⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠⚠

Reply:
Why